applications/DEMApplication/python_scripts/dem_default_input_parameters.py

```diff
@@ -77,6 +77,13 @@ def GetDefaultInputParameters():
             "CoordinationNumber"               : 10,
             "LocalCoordinationNumberOption"    : false,
             "GlobalCoordinationNumberOption"    : true,
+            "inner_mesh_diameter" : 0.00015,
```

Comment:
We need to discuss this @salvalatorre . Phone will work. It's just that it can't be understood. I mean: inner of what? Is it related to SP? If not, how could this be used by a generic DEM user?

Reply:
Done!